AEPIdentity/Tests/MobileIdentitiesTests.swift

```diff
@@ -30,18 +31,17 @@ class MobileIdentitiesTests: XCTestCase {
     ]
 
     private func buildIdentitySharedState() -> [String: Any] {
-        var identitySharedState = [String: Any]()
-        identitySharedState[IdentityConstants.EventDataKeys.VISITOR_ID_ECID] = "test-ecid"
-
         let customIdOne = CustomIdentity(origin: "origin1", type: "type1", identifier: "id1", authenticationState: .authenticated)
         let customIdTwo = CustomIdentity(origin: "origin2", type: "type2", identifier: "id2", authenticationState: .loggedOut)
         let customIdThree = CustomIdentity(origin: "origin3", type: "DSID_20915", identifier: "test-advertisingId", authenticationState: .loggedOut)
-
-        identitySharedState[IdentityConstants.EventDataKeys.VISITOR_IDS_LIST] = [customIdOne, customIdTwo, customIdThree].map({$0.asDictionary()})
-        identitySharedState[IdentityConstants.EventDataKeys.ADVERTISING_IDENTIFIER] = "test-advertisingId"
-        identitySharedState[IdentityConstants.EventDataKeys.PUSH_IDENTIFIER] = "test-pushid"
-
-        return identitySharedState
+        
+        var properties = IdentityProperties()
```

Comment:
These tests were passing correctly before and after my changes to the public API. However, I don't feel comfortable about how this setup assumes the structure of the VISITOR_IDS_LIST. Instead, I feel it's safer to use the actual IdentityProperties to build the event data.